lang/de.json

```diff
@@ -279,11 +279,15 @@
         ],
         [
             "👾 ALIEN",
-            "• `dmg` | `damage` : Krtisiche Verletzungen durch Schaden.\n• `s`, `synth` | `synthetic` : Kritische Verletzungen von Androiden.\n• `x` | `xeno` : Kritische Verletzungen bei Xenomorphen.\n• `m` | `mental` : Dauerhafte mentale Traumata."
+            "• `dmg` | `damage` : Kritische Verletzungen durch Schaden.\n• `s`, `synth` | `synthetic` : Kritische Verletzungen von Androiden.\n• `x` | `xeno` : Kritische Verletzungen bei Xenomorphen.\n• `m` | `mental` : Dauerhafte mentale Traumata."
         ],
         [
             "🌟 Coriolis: Der Dritte Horizont",
             "• `dmg` | `damage` : Kritische Verletzungen durch Schaden.\n• `at` | `atypical` : Kritische Verletzungen durch atypischen Schaden."
+        ],
+        [
+            "🦋 Vaesen",
+            "• `dmg` | `damage` : Physical critical injuries.\n• `m` | `mental` : Mental critical injuries."
```

Comment:
@mprangenberg Could you add your translation to this line please ? :-)

Reply:
```suggestion
            "• `dmg` | `damage` : Kritische Verletzungen durch physischen Schaden.\n• `m` | `mental` : Kritische Verletzungen durch mentalen Schaden."
```